dist/s_void

```diff
@@ -66,20 +66,54 @@ func_ok()
         -e '/int __time_value_validate_parent_stable$/d' \
         -e '/int __tree_walk_skip_count_callback$/d' \
         -e '/int __txn_rollback_to_stable_custom_skip$/d' \
+        -e '/int __ut_ckpt_add_blkmod_entry$/d' \
+        -e '/int __ut_ovfl_discard_verbose$/d' \
+        -e '/int __ut_ovfl_discard_wrapup$/d' \
         -e '/int __win_terminate$/d' \
         -e '/int __wt_block_compact_end$/d' \
         -e '/int __wt_block_compact_start$/d' \
         -e '/int __wt_block_disagg_manager_create$/d' \
-        -e '/int __wt_block_disagg_manager_size$/d' \
-        -e '/int __wt_block_disagg_read$/d' \
         -e '/int __wt_block_manager_size$/d' \
         -e '/int __wt_block_tiered_load$/d' \
         -e '/int __wt_block_write_size$/d' \
         -e '/int __wt_btcur_skip_page$/d' \
         -e '/int __wt_buf_catfmt$/d' \
         -e '/int __wt_buf_fmt$/d' \
-        -e '/int __wt_count_birthmarks$/d' \
         -e '/int __wt_conn_prefetch_clear_tree$/d' \
+        -e '/int __wt_count_birthmarks$/d' \
+        -e '/int __wt_curhs_search_near_after$/d' \
+        -e '/int __wt_curhs_search_near_before$/d' \
+        -e '/int __wt_epoch$/d' \
+        -e '/int __wt_errno$/d' \
+        -e '/int __wt_futex_wait/d' \
+        -e '/int __wt_futex_wake/d' \
+        -e '/int __wt_get_vm_pagesize$/d' \
+        -e '/int __wt_lex_compare_skip$/d' \
+        -e '/int __wt_logop_read$/d' \
+        -e '/int __wt_logop_unpack$/d' \
+        -e '/int __wt_logrec_read$/d' \
+        -e '/int __wt_once$/d' \
+        -e '/int __wt_session_breakpoint$/d' \
+        -e '/int __wt_set_return_func$/d' \
+        -e '/int __wt_spin_init$/d' \
+        -e '/int __wt_spin_trylock$/d' \
+        -e '/int __wt_stat_connection_desc$/d' \
+        -e '/int __wt_stat_dsrc_desc$/d' \
+        -e '/int __wt_stat_session_desc/d' \
+        -e '/int __wt_struct_pack_checkpoint_start$/d' \
+        -e '/int __wt_struct_unpack_backup_id$/d' \
+        -e '/int __wt_struct_unpack_checkpoint_start$/d' \
+        -e '/int __wt_struct_unpack_col_modify$/d' \
+        -e '/int __wt_struct_unpack_col_put$/d' \
+        -e '/int __wt_struct_unpack_col_remove$/d' \
+        -e '/int __wt_struct_unpack_col_truncate$/d' \
+        -e '/int __wt_struct_unpack_prev_lsn$/d' \
+        -e '/int __wt_struct_unpack_row_modify$/d' \
+        -e '/int __wt_struct_unpack_row_put$/d' \
+        -e '/int __wt_struct_unpack_row_remove$/d' \
+        -e '/int __wt_struct_unpack_row_truncate$/d' \
+        -e '/int __wt_struct_unpack_txn_timestamp$/d' \
+        -e '/int __wt_txn_read_upd_list$/d' \
         -e '/int __wti_block_disagg_checkpoint_start$/d' \
```

Comment:
Cannot leave a comment for all these calls, but why did you leave all the `__wti_block_disagg_` calls untouched?